import/index_java/src/org/vufind/index/ConfigManager.java

```diff
@@ -166,25 +172,30 @@ public Ini loadConfigFile(String filename)
      * @param filename configuration file name
      * @param section section name within the file
      */
-    public Map<String, String> getSanitizedConfigSection(String filename, String section)
+    public Map<String, String> getConfigSection(String filename, String section)
     {
-        Map<String, String> retVal = getConfigSection(filename, section);
+        Map<String, String> retVal = getRawConfigSection(filename, section);
```

Comment:
I think this might be more clear if we rename the variables: retVal --> rawSection and retValCopy --> retVal. Only one of these is actually a return value.